src/test/java/org/fundacionjala/coding/yerel/KataBankOCRTest.java

```diff
@@ -0,0 +1,117 @@
+package org.fundacionjala.coding.yerel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * test case for KataBanck.
+ */
+public class KataBankOCRTest {
+    private KataBankOCR kataBankOCR;
+
+    /**
+     * instance of class KataBankORC.
+     */
+    @Before
+    public void setUp() {
+        kataBankOCR = new KataBankOCR();
+    }
+
+    /**
+     * verificate if a number 000000000.
+     */
+    @Test
+    public void accountNumber() {
+        String number = " _  _  _  _  _  _  _  _  _ \n"
+                + "| || || || || || || || || |\n"
+                + "|_||_||_||_||_||_||_||_||_|\n";
+        assertEquals("000000000", kataBankOCR.accountNumber(number));
+    }
+
+    /**
+     * test case for validate 333333333.
+     */
+    @Test
+    public void accountNumberTwo() {
+        String number = " _  _  _  _  _  _  _  _  _ \n"
+                + " _| _| _| _| _| _| _| _| _|\n"
+                + " _| _| _| _| _| _| _| _| _|\n";
+        assertEquals("333333333", kataBankOCR.accountNumber(number));
+    }
+
+    /**
+     * test case for validate 123456789.
+     */
+    @Test
+    public void accountNumberThree() {
+        String number = "    _  _     _  _  _  _  _ \n"
+                + "  | _| _||_||_ |_   ||_||_|\n"
+                + "  ||_  _|  | _||_|  ||_| _|\n";
+        assertEquals("123456789", kataBankOCR.accountNumber(number));
+    }
+
+    /**
+     * test case for validate number 000000000.
+     */
+    @Test
+    public void convertNumber() {
+        List<StringBuilder> listNumber = new ArrayList<>();
+        listNumber.add(new StringBuilder(" _ | ||_|"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal " _ | ||_|" 17 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)